src/lib/Mapper/PagerDataMapperInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\Platform\Search\Mapper;
+
+use Pagerfanta\Pagerfanta;
+
+interface PagerDataMapperInterface
+{
+    public function map(Pagerfanta $pagerfanta);
```

Comment:
could we add return typehint (`array`) as well?

Reply:
sure, added